admin-portal/package.json

```diff
@@ -3,9 +3,9 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@asgardeo/auth-react": "^1.1.11",
+    "@asgardeo/auth-react": "^1.1.16",
     "@asgardeo/auth-spa": "^0.4.4",
-    "@asgardeo/token-exchange-plugin": "^0.1.3",
+    "@asgardeo/token-exchange-plugin": "0.1.3",
```

Comment:
Had to stick the version due to the [breaking changes](https://github.com/asgardeo/asgardeo-auth-spa-token-exchange/commit/0840597ace6964c42434873c36ec7bb42d33544f) in `0.1.4`.

@yathindrakodithuwakku please advice the team on the migration.